salt/cli/batch.py

```diff
@@ -103,7 +103,7 @@ def __update_wait(self, wait):
         if i:
             del wait[:i]
 
-    def run(self):
+    def run(self, safe_batch=False):
```

Comment:
We should document what this kwarg does. Is the intention here that this will be a string which contains a number that will later be cast to an int?